runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1718,7 +1718,7 @@ def GotoVariableswinOrCreateIt()
     win_gotoid(varwin)
   endif
 
-  if running && bufwinid(varbufname) > 0
+  if running && varwin > 0
```

Comment:
If I understand the logic here, when this is executed, there is **always** a `:Vars` window.

And the `varwin` variable is stale if the window was created.

Reply:
True. Once you get there a` :Var` window is there. We don't need the `&&` condition.